client/src/utils/plot.js

```diff
@@ -1,7 +1,7 @@
 import * as d3 from 'd3';
 import Debug from 'debug';
 import { legendColor } from 'd3-svg-legend';
-import processLatexString from './stringUtils';
+import createLatexSvgFromString from './latex';
```

Comment:
Had a feeling you might bring that up. `latex2svg` is actually the name of a very popular function in a library elsewhere that has to do with. I used it at one point. So while it is fine to go with your suggestion in the other location, it doesn't feel right here, because it was used here at one point. The other function name makes more sense personally. https://www.npmjs.com/package/latex2svg

Reply:
Feel free to rename it in one of your commits, but I disagree with the renaming in this location.